smoke/esm/it.sh

```
@@ -4,77 +4,264 @@ projectBase=$(pwd)
-#cat <<EOF > $packageDir/.npmrc
-#; .npmrc
-#enable-pre-post-scripts=true
-#//localhost:4873/:_authToken=$token
-#@fireproof:registry=http://localhost:4873
-#EOF
-
-#(cd $packageDir &&
-#	(npm unpublish --force || true) &&
-#         npm publish --no-git-checks)
+check_registry_health() {
+  local registry_status=$(curl --retry 3 --retry-max-time 10 --retry-all-errors -s -o /dev/null -w "%{http_code}" http://localhost:4873/)
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)